addons/udes_security/models/res_groups.py

```diff
@@ -13,9 +13,11 @@ class Groups(models.Model):
     @api.multi
     def write(self, values):
         group_trusted_user = self.env.ref('udes_security.group_trusted_user')
+        group_debug_user = self.env.ref('udes_security.group_debug_user')
```

Comment:
I would suggest to have list of groups in a constant of the file and loop over it.